File: src/main/java/com/muxiu1997/mxrandom/network/container/ContainerConfigLargeMolecularAssembler.java
package com.muxiu1997.mxrandom.network.container;

import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.inventory.Container;

import com.muxiu1997.mxrandom.metatileentity.MTELargeMolecularAssembler;

public class ContainerConfigLargeMolecularAssembler extends Container {

    public final MTELargeMolecularAssembler LMA;

    public ContainerConfigLargeMolecularAssembler(MTELargeMolecularAssembler LMA) {
        this.LMA = LMA;
    }

    @Override
    public boolean canInteractWith(EntityPlayer player) {
        return true;
    }
}

File: src/main/kotlin/com/muxiu1997/mxrandom/network/container/ContainerConfigLargeMolecularAssembler.kt
package com.muxiu1997.mxrandom.network.container

import com.muxiu1997.mxrandom.metatileentity.MTELargeMolecularAssembler
import net.minecraft.entity.player.EntityPlayer
import net.minecraft.inventory.Container

class ContainerConfigLargeMolecularAssembler(val LMA: MTELargeMolecularAssembler) : Container() {
  override fun canInteractWith(player: EntityPlayer?): Boolean {
    return true
  }
}
